tools/monitor/model.py

```diff
@@ -50,3 +55,121 @@ def create_from_frame(frame):
     @staticmethod
     def is_download_frame(frame):
         return type(frame) in [FileSendSuccessFrame, FileSendErrorFrame]
+
+
+class MemoryFrameView:
+    def __init__(self, correlation_id, sequence_number):
+        self.correlation_id = correlation_id
+        self.chunk = sequence_number
+        self.is_success = True  # used by GUI
+
+    @staticmethod
+    def create_from_frame(frame):
+        if not MemoryFrameView.is_memory_frame(frame):
+            return None
```

Comment:
to już w sumie jest sprawdzane wyżej?

Reply:
To celowo - jakby ktoś wziął tę klasę spoza monitora i chciał użyć, to zadziała tylko z MemoryFrame.
Niestety to python 2 i nie ma MyPy, który to by ogarnął. :)